src/style.css

```diff
@@ -77,3 +77,7 @@
   left: -20px;
   border-style: none;
 }
+
```

Comment:
### COMMENTED CODE BLOCK

- [ ] Please, let's remove this commented [code](https://github.com/fuadmoin/TODOLIST/blob/9d0ef2bab6e9e84a55e6ad7f3a3241e107fcbb7d/src/style.css#L32) to keep our codebase clean.